packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -92,13 +92,15 @@ export class Dropdown extends BaseComponent<IDropdownProps, IDropdownState> {
           onClick={ this._onDropdownClick }
           aria-expanded={ isOpen ? 'true' : 'false' }
           role='combobox'
+          aria-live={ disabled || isOpen ? 'off' : 'assertive' }
```

Comment:
If there's another screen reader read out at the same time, the option text will be skipped and wont read out anything.